py3dtiles/gltf.py

```diff
@@ -207,8 +205,8 @@ def compute_header(binVertices, nVertices, bb, transform,
             'byteOffset': sum(sizeVce[0:i]),
             'componentType': 5126,
             'count': nVertices[i],
-            'max': [bb[i][0][1], bb[i][0][2], bb[i][0][0]],
-            'min': [bb[i][1][1], bb[i][1][2], bb[i][1][0]],
+            'min': [bb[i][0][0], bb[i][0][1], bb[i][0][2]],
```

Comment:
Was the original a bug ? What does this subtle yet probably
mandatory modification correct/improve ?

Reply:
It's weird that github finds a modification here since I proposed this change in #3 which has already been merged into the Tiler branch. @EricBoix we may need to look into this tomorrow.

Reply:
The problem is that this branch was not rebased on the Tiler branch.
Require this rebase.